WordPress/gutenberg
# Query Total: Add Border Support

## What?

Add Border support to the `Query Total` block.

Part of https://github.com/WordPress/gutenberg/issues/43247

## Why?
`Query Total` block is missing Border support.

## How?

Add the border support in block.json.

## Testing Instructions

- Go to Global Styles Settings. ( Under Appearance > Editor > Styles > Edit styles > Blocks )
- Make sure that `Query Total` block's border is Configurable via Global Styles.
- Edit Archives Template &  Add `Query Total` block and Apply the border Styles.
- Verify that `Query Total` block styles take precedence over global Styles.
- Verify that `Query Total` block borders display correctly in both the Editor and Frontend.

## Screenshots or Screencast 

https://github.com/user-attachments/assets/05a0b27e-38c1-4b3c-ac7e-ccf532c658cf
